<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vvSlamConfigurationDialog</class>
 <widget class="QDialog" name="vvSlamConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Advance (leave default if you don't know)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="2">
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Mapping</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0,0" rowminimumheight="0,0,0">
         <item row="1" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>ICP frequence</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string># Itertations</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="Mapping_MaxIter">
           <property name="value">
            <number>15</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="Mapping_IcpFrequence">
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox_10">
           <property name="title">
            <string>Edges</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="1" column="0">
             <widget class="QLabel" name="label_35">
              <property name="text">
               <string>Min neighbor rejection</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="MappingMinimalLineNeighborRejection">
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string># Neighbor</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="Mapping_LineDistance_k">
              <property name="value">
               <number>15</number>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="Mapping_LineDistance_factor">
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
              <property name="value">
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>factor1</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>max distance</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="Mapping_Line_Max_Distance">
              <property name="value">
               <double>0.200000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_36">
              <property name="text">
               <string>max dist inlier rejection</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="MappingMaxDistanceInlierRejection">
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.300000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QGroupBox" name="groupBox_9">
           <property name="title">
            <string>Planes</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="1">
             <widget class="QSpinBox" name="Mapping_PlaneDistance_k">
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="Mapping_PlaneDistance_factor1">
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
              <property name="value">
               <double>30.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string># Neighbor</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="Mapping_PlaneDistance_factor2">
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
              <property name="value">
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>factor2</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>factor1</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>max distance</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="Mapping_Plane_Max_Distance">
              <property name="value">
               <double>0.040000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Keypoint</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="3">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>min edges sinus angle</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string># Planes / scane line</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string># Edges / scane line</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="Keypoint_MaxPlanarsPerScanLine">
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>200</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="Keypoint_MaxEdgePerScanLine">
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>200</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Max planes sinus angle</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="Keypoint_MinDistanceToSensor">
           <property name="value">
            <double>3.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QDoubleSpinBox" name="Keypoint_EdgeSinAngleThreshold">
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.850000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QDoubleSpinBox" name="Keypoint_PlaneSinAngleThreshold">
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QCheckBox" name="FastSlam">
           <property name="text">
            <string>Fast Slam</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>Fast Slam</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Min distance to sensor</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>min edges depth gap</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QDoubleSpinBox" name="Keypoints_EdgeDepthGapThreshold">
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
           <property name="value">
            <double>0.020000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>EgoMotion</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="1">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string># Iterations</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>Edges</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="2" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>factor1</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string># Neighbor</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="EgoMotion_LineDistance_k">
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="EgoMotion_LineDistance_factor">
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
              <property name="value">
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>max distance</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="EgoMotion_Line_Max_Distance">
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_34">
              <property name="text">
               <string>Min neighbor rejection</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="EgoMotionMinimalLineNeighborRejection">
              <property name="value">
               <number>2</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="3" colspan="2">
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string>Planes</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="2" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>factor2</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="EgoMotion_PlaneDistance_factor1">
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
              <property name="value">
               <double>50.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="EgoMotion_PlaneDistance_k">
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="EgoMotion_PlaneDistance_factor2">
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
              <property name="value">
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string># Neighbor</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>factor1</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>max distance</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="EgoMotion_Plane_Max_Distance">
              <property name="value">
               <double>0.040000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="EgoMotion_MaxIter">
           <property name="value">
            <number>15</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>ICP frequence</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="EgoMotion_IcpFrequence">
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>General</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_11">
         <item row="0" column="1">
          <widget class="QSpinBox" name="NbVoxel">
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="AngleResolution">
           <property name="value">
            <double>0.400000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Nb Voxel (n*n*n)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>L-M lamba 0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="MaxDistanceICP">
           <property name="value">
            <double>20.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="Lambda0">
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Angle Resolution</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>Max distance ICP</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>L-M Lambda ratio</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="LambdaRatio">
           <property name="value">
            <double>1.500000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Slam</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="allFrames">
        <property name="text">
         <string>All frames</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="End">
        <property name="maximum">
         <number>200000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="Start">
        <property name="maximum">
         <number>200000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="frameRange">
        <property name="text">
         <string>Frames</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLabel" name="label_39">
        <property name="text">
         <string>Max angle acceleration (degree.s-2)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="currentFrame">
        <property name="text">
         <string>Current frame</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QCheckBox" name="MotionModelCheckbox">
        <property name="text">
         <string>Motion Model</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_38">
        <property name="text">
         <string>Max velocity acceleration (m.s-2) </string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QDoubleSpinBox" name="AngleAccelerationSpinBox">
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>540.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QCheckBox" name="MotionModelAndGPSCheckbox">
        <property name="text">
         <string>Motion Model + GPS</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QDoubleSpinBox" name="VelocityAccelerationSpinBox">
        <property name="maximum">
         <double>500.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>9.810000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QLabel" name="label_37">
        <property name="text">
         <string>Motion model</string>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="QCheckBox" name="UndistortionCheckBox">
        <property name="text">
         <string>Frame Undistortion</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>vvSlamConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>598</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>vvSlamConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>604</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
